src/utils/GraphhopperUtils.js

```diff
@@ -305,6 +291,33 @@ async function fetchRoutes(end: string, start: string, departureTimeDateNow: str
     );
   }
 
+  return routes;
+}
+
+/*
+* Return walkingRoute from the Graphhopper walking service given the parameters
+* walkingRoute contains the shortest possible walking path
+* @param end
+* @param start
+* @returns Object
+*/
+async function fetchWalkingRoute(end: string, start: string): Object {
+  let walkingRoute;
```

Comment:
probably better to just have this be a `const` and `return` in both cases of the `if/else` instead, but up to you!